lib/Language/Haskell/Stylish/Step/Imports.hs

```diff
@@ -103,6 +119,27 @@ data LongListAlign
     | Multiline -- multiline
     deriving (Eq, Show)
 
+data Pattern = Pattern
```

Comment:
Haddock?

Reply:
Happy to add haddocks here—I didn't write any because the rest of the codebase didn't seem to use haddock comments.

Reply:
"Be the change you want to see in the world" or something? :)